Documentation/gettingstarted/k8s-install-external-etcd.rst

```diff
@@ -69,7 +70,7 @@ key and certificate of etcd:
 
 .. code:: bash
 
-   kubectl create secret generic -n kube-system cilium-etcd-secrets \
+    kubectl create secret generic -n kube-system cilium-etcd-secrets \
```

Comment:
nit: was this leading space intentional?

Reply:
Yeah, the indentation was not correct earlier.